### ISSUE TITLE
python chunk inserted even when R chunk is already in file

### ISSUE THREAD

**Issue by jjallaire** [rstudio/rstudio#9678]
For this markdown in qmd mode:

````markdown
---
title: "kable"
format:
  html:
    theme: default
---

Here is a table:

```{r, echo=FALSE}
knitr::kable(pressure)
```

Here is some text:

````

The insert chunk command inserts a python chunk.

**Comment by jjallaire** [comment]
Fixed with https://github.com/rstudio/rstudio/commit/db6934360d136bdd8ca7d97c9cc6267a748f1b3b

Previously we would insert a python chunk if the existing R chunk add additoinal attributes (e.g. ```{r, echo=FALSE}). Now, we look through the file for the first executable chunk and match it's language. If no executable chunks are found then we insert an R chunk.

**Comment by astayleraz** [comment]
Verified this was fixed in version 2021-07.0.285-2 and added an automated test for this case. Closing issue.
